src/content/docs/dev/general-message-passing/sui/gmp-tutorial.mdx

```diff
@@ -0,0 +1,918 @@
+import { Callout } from "../../../../../components/callout";
+
+# How to Send Cross-Chain Messages from Sui to EVM
+
+This guide walks you through creating a simple yet powerful application that sends messages between Sui and [Ethereum-compatible](https://ethereum.org/en/developers/docs/evm/) blockchains.
+
+In this beginner-friendly tutorial, you'll learn how General Message Passing (GMP) lets your applications communicate across different blockchains, opening up exciting possibilities for your Web3 projects.
+
+What You'll Build
+
+A cross-chain messaging application that:
+- Sends messages from Sui to any EVM chain
+- Receives messages from EVM chains back to Sui
+- Demonstrates the core concepts of blockchain interoperability
+
+## Prerequisites
+
+Before starting, you'll need these tools installed:
+- [Rust](https://www.rust-lang.org/tools/install) - Programming language
+- [Sui CLI](https://docs.sui.io/build/install) - Command line interface
+- [Sui Move](https://docs.sui.io/build/move) - Smart contract language
+- [Sui JS](https://www.npmjs.com/package/@mysten/sui) - JavaScript SDK
+
+## Setup
+
+We'll use Sui's TypeScript SDK to build and deploy our `move` contract, with `npm` managing our dependencies. Let's get started with setting up your Sui development environment:
+
+```bash
+sui move new sui-gmp
+```
+
+This will create a new directory called `sui-gmp` with the following structure:
+
+1. `Move.toml`    
+1. `sources`      
+1. `tests`
+
+You can the run `npm init -y` to add a `package.json` file to the project.
+
+### Move Config
```

Comment:
```suggestion

```

Reply:
why was this removed? The `move.toml` is a significant part worth addressing imo